README.md

```diff
@@ -10,6 +9,73 @@ Please refer to our [readthedocs documentation](https://deeppavlov-agent.readthe
 - Jenkins Tests Status on dev branch: [![Build Status](http://lnsigo.mipt.ru:8080/buildStatus/icon?job=assistant%2Fdev)](http://lnsigo.mipt.ru:8080/job/dp-multibranch/job/dev/)
 
 
+# Distributions
+
+We've already included two distributions: Deepy and a full-sized Dream chatbot.
+
+
+# Quick Start
+
+### Deploying via proxy
+The easiest way to try out Dream is to deploy it via proxy.
+This way all the requests will be redirected to DeepPavlov API, so you don't have to use any local resources.
+
+1. Clone the repo
+
+    ```git clone https://github.com/deepmipt/dream```
+
+2. Run
+
+    ```docker-compose -f docker-compose.yml -f dev.yml -f proxy.yml up --build```
+    
+    `dev.yml` includes volume bindings for easier debugging;
+
+    `proxy.yml` is a list of proxied containers. See [proxy usage](#proxy-usage).
+
+
+### Deploying components locally
+
+Any components which are not included in `proxy.yml` will be deployed locally.
+
+
+# Proxy usage
+
+If your deployment resources are limited, you can replace containers with their proxied copies hosted by DeepPavlov.
+To do this, override those container definitions inside `proxy.yml`, e.g.:
+```
+convers-evaluator-annotator:
+  command: ["nginx", "-g", "daemon off;"]
+  build:
+    context: dp/proxy/
+    dockerfile: Dockerfile
+  environment:
+    - PROXY_PASS=lnsigo.mipt.ru:8004
+    - PORT=8004
+```
+and include this config in your deployment command:
+```
+docker-compose -f docker-compose.yml -f proxy.yml up
+```
+By default, `proxy.yml` contains all available proxy definitions.   
+
```

Comment:
please add section run locally full dream 
Huge important info that it consume a lot of resources.